src/D2L.Bmx/Program.cs

```diff
@@ -131,6 +132,7 @@
 writeCommand.SetHandler( ( InvocationContext context ) => RunWithErrorHandlingAsync( context, () => {
 	var consolePrompter = new ConsolePrompter();
 	var config = new BmxConfigProvider().GetConfiguration();
+	var parser = new FileIniDataParser();
```

Comment:
well maybe I should just put it directly in and not create a var. I was thinking maybe we want to configure it more before passing in but I don't think so now